addons/stock/wizard/stock_quantity_history.py

```diff
@@ -35,4 +35,20 @@ def open_table(self):
         else:
             self.env['stock.quant']._merge_quants()
             self.env['stock.quant']._unlink_zero_quants()
-            return self.env.ref('stock.quantsact').read()[0]
+            action = self.env.ref('stock.quantsact').read()[0]
+            context = {
+                'search_default_internal_loc': 1,
+                'inventory_mode': True,
```

Comment:
huh
now i'm thinking we shouldn't set the inventory_mode in the action xml but only force it on this wizard + the quantity on hands buttons